static/docs/use-cases/data-registry.md

```diff
@@ -0,0 +1,166 @@
+# Data Registry
+
+We developed the `dvc get`, `dvc import`, and `dvc update` commands with the aim
```

Comment:
we will need to do a `programatic access` or `access from code` or `access from Python` h2 section here with a reference to python API. Please, create a ticket for this.

Reply:
OK: #779